frontend/utils.js

```diff
@@ -29,9 +29,9 @@ function removeEndingSlash(val) {
   return val
 }
 
-export const API_BASE_URL =
-  removeEndingSlash(getConfig().publicRuntimeConfig.API_BASE_URL) ||
-  'https://pennclubs.com/api'
+export const DEFAULT_SITE_ORIGIN = process.env.NODE_ENV === 'production' ? 'https://pennclubs.com' : `http://localhost:${process.env.PORT || 3000}`
+export const SITE_ORIGIN = removeEndingSlash(getConfig().publicRuntimeConfig.API_BASE_URL) || DEFAULT_SITE_ORIGIN
```

Comment:
Do we still need to check `API_BASE_URL` with the above changes?

Reply:
We don't, do you think we should just remove it?

Reply:
I think we could. We would lose the ability to set an arbitrary url for the backend, but I don't know if we need to